### PR TITLE
Use separate spreadsheet for shared data and one for the list of responses spreadsheets

### PR DESCRIPTION
**Pull request by AndrewLester**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 venv/
```
Comment (ErikBoesen) [inline .gitignore]: Can probably remove `venv` from this list since this isn't a Python program.

**Comment by ErikBoesen** [comment]
Didn't we decide this is too slow?

**Comment by AndrewLester** [comment]
> Didn't we decide this is too slow?

It is slower, but with the blur it isn't as noticeable. I just timed it and it's like 2 seconds this way and 1 second normally.

**Comment by ErikBoesen** [comment]
I really don't think it's worth doubling our load time just so devs can add a line to a spreadsheet instead of a JS file.

**Comment by AndrewLester** [comment]
> I really don't think it's worth doubling our load time just so devs can add a line to a spreadsheet instead of a JS file.

True, but I feel like it would make it easier for a non dev to use in case someone isn't chosen. If I were to write an instruction sheet on how to add another year, I think someone like Mr. Knight could handle it too.

An alternative would be to add the current spreadsheets to the code by default, and if there is a dev working on it they can just extend the code. If there is no dev (a sheet ID is missing in the code), it will look in the spreadsheet.

**Comment by ErikBoesen** [comment]
I guess that's fair, but are we going to need to do that anytime soon? How about commenting out the spreadsheet loading code and then it can be reenabled when we have a non-dev that needs to handle it, like next year? That way we don't double people's load time all this year before the change actually starts having any merit.

Beyond that, I'm not entirely sure that we can fully remove developers from the project. Someone still needs to make sure that all the columns on the spreadsheets are in the right order, and change privacy settings, etc.

**Comment by AndrewLester** [comment]
I didn't comment it out exactly, but I made the spreadsheet loading code skip itself if the year data being requested is already hardcoded.

**Comment by ErikBoesen** [comment]
Very good move.

**Pull request merged**